.github/workflows/Nightly.yml

```diff
@@ -87,6 +88,21 @@ jobs:
           enableCrossOsArchive: true
           restore-keys: |
             external-data-v1-
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v5
+        id: cpy
+        with:
+          python-version: 3.11
```

Comment:
Should this use Python 3.12 or 3.13? I was looking at the [status of Python versions page](https://devguide.python.org/versions/) recently and realized that Python 3.11 is starting to get old.

Reply:
It is good enough for a couple of years.